index.html

```diff
@@ -253,9 +253,9 @@ <h2>Contact me</h2>
             <p>If you have an application you are interested in developing, a feature that you
                  need built or a project that needs coding. I’d love to help with it </p>
             <form action="https://formspree.io/f/mknyyevq" method="post" id="contact-form">
-                <input type="text" placeholder="Rehema" maxlength="30" required>
-                <input id="email" type="email" name="email" placeholder="mwakabayah@gmail.com" required>
-                <textarea cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+                <input type="text" name="name" value="" placeholder="Rehema" maxlength="30" required>
+                <input id="email" name="email" value="" type="email" name="email" placeholder="mwakabayah@gmail.com" required>
+                <textarea cols="30" rows="10" maxlength="500" name="message" value="message" placeholder="Write your message here" required></textarea>
```

Comment:
- [ ] [optional] It would be nice to use the default values as seen in the Figma design given, you could also use a placeholder like `enter your name/password` which is more descriptive to what the user is suppose todo.